src/components/wallet/methods/depositAsset.ts

```diff
@@ -61,147 +46,129 @@ export default async function depositAsset(
       `Received WEB_AUTH_ENDPOINT from TOML: ${toml.WEB_AUTH_ENDPOINT}`
     )
     this.logger.instruction(
-      `Received TRANSFER_SERVER from TOML: ${toml.TRANSFER_SERVER}`
+      `Received TRANSFER_SERVER_SEP0024 from TOML: ${toml.TRANSFER_SERVER_SEP0024}`
```

Comment:
We updated SEP24 at some point to use the key TRANSFER_SERVER_SEP0024 instead of TRANSFER_SERVER so that it doesn't conflict with SEP6's TRANSFER_SERVER key